cli/src/cli_util.rs

```diff
@@ -1290,6 +1290,11 @@ impl WorkspaceCommandHelper {
             .expect("parse error should be confined by WorkspaceCommandHelper::new()")
     }
 
+    pub fn format_short_change_id(&self) -> TemplateRenderer<'_, Commit> {
```

Comment:
nit: `format_()` sounds like a function that returns formatted object. maybe rename to `short_change_id_template()`?